src/neo/Network/P2P/Message.cs

```diff
@@ -26,7 +26,7 @@ public class Message : ISerializable
 
         public int Size => sizeof(MessageFlags) + sizeof(MessageCommand) + _payload_compressed.GetVarSize();
 
-        public static Message Create(MessageCommand command, ISerializable payload = null)
+        public static Message Create(MessageCommand command, ISerializable payload = null, bool tryCompression = false)
```

Comment:
Maybe we can decide whether to compress or not through `command`.

Reply:
why not by data size.

Reply:
Because some type of messages are definitely not compressible, regardless of size. Such as `Inv` messages.

Reply:
I will use the command, give me a couple of days

Reply:
Done